components/camel-undertow/src/main/java/org/apache/camel/component/undertow/UndertowComponent.java

```diff
@@ -435,4 +476,8 @@ public String getAllowedRoles() {
     public void setAllowedRoles(String allowedRoles) {
         this.allowedRoles = allowedRoles;
     }
+
+    public UndertowSecurityProvider getSecurityProvider() {
```

Comment:
Add setter so you can set this explicit

Reply:
I can do it also this way, in that case please ignore my previous comment